src/main/java/org/opentripplanner/transit/service/DefaultTransitService.java

```diff
@@ -78,6 +79,11 @@ public Collection<Agency> getAgencies() {
     return this.transitModel.getAgencies();
   }
 
+  @Override
+  public Optional<Agency> getAgencyById(FeedScopedId id) {
```

Comment:
If you get something by ID the method should throw an exception not return an optional, if you search for something you should use optonal:

https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/CODE_CONVENTIONS.md#methods

Reply:
Ah, I was looking for `findAgency`! Do we want to add `ById`?

Reply:
I've changed the name in f7a983b7db86c8b08f84b04e08b5161eeeb3d63c.